nodeEditor/src/diagram/graphFrame.ts

```diff
@@ -677,12 +740,17 @@ export class GraphFrame {
                 return;
             }
             elementsArray = Array.from(this._inputPortContainer.childNodes);
+            const indexInContainer = this._frameInPorts.findIndex(framePort => framePort === nodePort);

```

Comment:
(same goes to the other findIndex calles) - is it guarantied that the element is inside the array? Or do you need to check that the index is 0 and up?

Reply:
This is guarantied. In fact the function call below this line also does a similar search with no bounds checking.